eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -17,7 +17,7 @@ parameters:
   centOSStream8Container: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream8
   centOSStream9Container: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream9
   fedora38Container: mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-38
-  ubuntu2004Container: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-20.04
+  ubuntu2204Container: mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-22.04
```

Comment:
What do you think about updating the comment above the container parameters to remind folks that when updating, corresponding changes are needed in the diff pipeline?

Reply:
I'm actually more inclined to put such a comment next to the `buildName` parameter for each job. I think it's more likely to be noticed there, and more relevant positioning.